dataprofiler/profilers/int_column_profile.py

```diff
@@ -70,6 +70,24 @@ def report(self, remove_disabled_flag: bool = False) -> dict:
         """
         return self.profile
 
+    @classmethod
+    def load_from_dict(cls, data):
+        """
+        Parse attribute from json dictionary into self.
+
+        :param data: dictionary with attributes and values.
+        :type data: dict[string, Any]
+
+        :return: Profiler with attributes populated.
+        :rtype: CategoricalColumn
+        """
+        profile = super().load_from_dict(data)
+        profile._load_hist_helper(data)
+        quantiles = data.pop("quantiles")
+        quantiles_dict = {int(key): quantiles[key] for key in quantiles.keys()}
```

Comment:
quantiles, I think, we will have to cast the keys to whatever the specific load_from_dict profiler is that were using ie:
floatprofiler will have to be cast to floats. This is because by default the keys are decoded as strings when loaded in